packages/components/src/submenu-popover/index.tsx

```diff
@@ -81,9 +83,9 @@ function useHasRightSpace( parentElement: HTMLElement | undefined, isVisible: bo
 }
 
 export function useSubmenuPopoverProps< T extends HTMLElement >(
-	options: { offset?: number } = {}
+	options: { offset?: number; flip?: boolean; resize?: boolean; inline?: boolean } = {}
```

Comment:
What do you think about defining those props `Partial< SubmenuPopoverProps >`, like below?

Reply:
Thanks @kozer - I think I see why it is a better approach as it allows us to provide explicit default values, is that right?

I made updates based on your suggestion!